<launch>
    <node pkg="actuator" type="actuator_slave.py" name="act_slave" output="screen" />
</launch>